libs/geoservices/core/ngx/src/lib/components/listable/submissions/county/county.component.spec.ts

```diff
@@ -8,9 +11,16 @@ describe('CountyComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ CountyListComponent ]
-    })
-    .compileComponents();
+      imports: [EnvironmentModule, HttpClientTestingModule],
+      declarations: [CountyListComponent],
+      providers: [
+        CountyListComponent,
```

Comment:
Components are *almost always* never providers. I can't think of a case where they would be, but there might be a reason to. To date, this should not be the case for any of our components.